vuln/npm/387.json

```diff
@@ -18,4 +18,4 @@
   "cvss_vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:C/C:H/I:N/A:N",
   "cvss_score": 8.6,
   "coordinating_vendor": ""
-}
+}
```

Comment:
Nit: why removing newline at the end of file?

Reply:
ugh, editor config :(
we should probably set up some linting for the project

Reply:
@lirantal There is already linting, run `npm it`. It could to be improved, though.